File: Java/hayaa.rpc.service/src/main/java/hayaa/rpc/common/protocol/MethodMessage.java
package hayaa.rpc.common.protocol;

import java.io.Serializable;
import java.util.HashMap;

public class MethodMessage  implements Serializable {
    private String interfaceName;
    private String method;
    private HashMap<String,Object> paramater;
    private String msgID;
    public String getMsgID() {
        return msgID;
    }

    public void setMsgID(String msgID) {
        this.msgID = msgID;
    }
    public String getInterfaceName() {
        return interfaceName;
    }

    public void setInterfaceName(String interfaceName) {
        this.interfaceName = interfaceName;
    }

    public String getMethod() {
        return method;
    }

    public void setMethod(String method) {
        this.method = method;
    }

    public HashMap<String, Object> getParamater() {
        return paramater;
    }

    public void setParamater(HashMap<String, Object> paramater) {
        this.paramater = paramater;
    }


}

File: C#/RPC/Hayaa.RPC.Componet/Protocol/MethodMessage.cs
using System;
using System.Collections.Generic;

namespace Hayaa.RPC.Componet
{
    [Serializable]
   public class MethodMessage
    {
        public String MsgID { set; get; }
        public String InterfaceName { set; get; }
        public String Method { set; get; }
        public List<RpcDataValue> Paramater { set; get; }

    }
}
